docs/examples/index.md

```diff
@@ -88,18 +81,18 @@ import { VscGraphLine } from "react-icons/vsc";
     icon={<IoEarth color="white" size={50}/>}
     language="python"
   />
-  <CardLink
-    label="Cloud Cron Quickstart"
-    href="python/examples/cron-starter"
-    description="Use DBOS to write a cron job in just six lines of code and host it in the cloud with a single command."
-    icon={<TbClock2 color="white" size={50}/>}
-    language="python"
-  />
   <CardLink
     label="Fault-Tolerant Checkout"
     href="typescript/examples/checkout-tutorial"
     description="Use DBOS durable workflows to build an online storefront that's resilient to any failure."
     icon={<MdOutlineShoppingCart color="white" size={50}/>}
     language="typescript"
   />
+  <CardLink
+    label="Hacker News Bot"
+    href="python/examples/hacker-news-bot"
+    description="Use DBOS to deploy a scheduled job that regularly searches Hacker News for comments about serverless computing and posts them to Slack."
+    icon={<FaHackerNews color="white" size={50} />}
+    language="python"
+  />
```

Comment:
Hey we can also add S3Mirror to this list.

Reply:
Let's write a docs page for it first